src/meshapi/tests/test_join_form.py

```diff
@@ -322,7 +322,7 @@ def test_member_moved_and_used_stripe_email_join_form(self):
         # and we will connect it to their old registration
         v_sub_3 = valid_join_form_submission.copy()
         v_sub_3["email"] = "jsmith+other@gmail.com"
-        v_sub_3["street_address"] = "153 Broome Street"
+        v_sub_3["street_address"] = "178 Broome Street"
```

Comment:
I had to change this because our API was consistently giving us 1000000 (a bad BIN) on 153 broome st. I guess they yeeted that BIN, but it's still working on NYC DOB website?
https://a810-bisweb.nyc.gov/bisweb/PropertyProfileOverviewServlet?boro=1&houseno=153&street=Broome+St&go2=+GO+&requestid=0

Not really sure what's going on.

Reply:
The test works with 178 Broome St